src/app/region.js

```diff
@@ -0,0 +1,188 @@
+import L from 'leaflet'
+import { uniq } from 'lodash'
+import { getTilesForBbox, getTileUrlSuffix, parseSegmentId } from '../lib/tiles'
+import { getRoute } from '../lib/valhalla'
+import { merge } from '../lib/geojson'
+import { setDataSource, getCurrentScene, setCurrentScene } from '../lib/tangram'
+import { fetchDataTiles } from './data'
+import store from '../store'
+import { startLoading, stopLoading, hideLoading } from '../store/actions/loading'
+import { setRouteError } from '../store/actions/route'
+
+const OSMLR_TILE_PATH = 'https://osmlr-tiles.s3.amazonaws.com/v0.1/geojson/'
+const host = 'routing-prod.opentraffic.io'
+
+function getSuffixes (bbox) {
+  const tiles = getTilesForBbox(bbox.min_lon, bbox.min_lat, bbox.max_lon, bbox.max_lat)
+  // Filter out tiles with level 2, no data for those
+  const downloadTiles = tiles.filter(function (tile) { return tile[0] !== 2 })
+  // Get suffixes of these tiles
+  const suffixes = downloadTiles.map(i => getTileUrlSuffix(i))
+  return suffixes
+}
+
+/**
+ * Goes through each coordinate, each line, each point, and removes (in place)
+ * latlngs that are outside bounding box
+ * It then removes array of points if empty, array of lines if empty, array of coordinates if empty,
+ * and finally, if that feature has no coordinates, it removes the feature from the array of features
+ *
+ * @param {array} features - geojson from OSMLR geometry tile
+ * @param {object} bounds - the latlngs of the bounding box
+ * @returns {array} - features that are within the bounding box
+*/
+
+function withinBbox (features, bounds) {
+  // We need to check the geometry.coordinates to check if they're within bounds
+  // If not within bound, remove entire feature from features
+
+  // Arbitrary number for buffer, edit if needed.
+  // Sometimes the endpoint of a line falls outside the bounding box which leads to
+  // but not by a lot. The buffer allows the line to be drawn.
+  const buffer = 0.0003
```

Comment:
it's also really common to define these variables at the top of a module. that way it's super easy to see what the parameters are and tweak a bunch of them at once, if necessary.